README.md

```diff
@@ -64,7 +64,7 @@ GitHub: [@Munsa1](https://github.com/munsa1)
 
 Contributions, issues, and feature requests are welcome!
 
-Feel free to check the [issues page](../../issues/).
+Feel free to check the [issues page](https://github.com/Munsa1/Awesome-Books/issues)
 
 ## Show your support
 
```

Comment:
- Add a descriptive README file. Provide information about your project.
- Kindly fix linters issues in your branch.

Reply:
thank you for the feedback